apps/server/src/modules/lti-tool/service/lti-tool.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import { LtiToolDO } from '@shared/domain/domainobject/ltitool.do';
+import { LtiToolRepo } from '@shared/repo';
+
+@Injectable()
+export class LtiToolService {
+	constructor(private readonly ltiToolRepo: LtiToolRepo) {}
+
+	public async findByClientIdAndIsLocal(clientId: string, isLocal: boolean): Promise<LtiToolDO | null> {
```

Comment:
Good job to capsulate the repo but hopefully this change will only exists for a short time